node/pkg/websocketfetcher/providers/mexc/utils.go

```diff
@@ -23,9 +23,16 @@ func ResponseToFeedDataList(response BatchResponse, feedMap map[string]int32) ([
 		if err != nil {
 			return feedDataList, err
 		}
+
+		volume, err := common.VolumeStringToFloat64(item.QuoteVolume)
```

Comment:
@nick-bisonai My guess that this is value to use for volume computation. You should be using base volume and not quote volume.

Reply:
if you see their documentation and see the example output, it is obvious they are using it in a opposite way. I checked the value and this is correct, I'll add comment about this

https://mexcdevelop.github.io/apidocs/spot_v3_en/#miniticker